specification.md

```diff
@@ -1855,6 +1860,20 @@ for this connection.
 This field is optional: if omitted the client SHOULD NOT use a client-side certificate.
 This field can be used to perform a client certificate revocation/rotation.
 
+##### OpAMPConnectionSettings.heartbeat_interval_seconds
+
+The Client should use the offered heartbeat interval to periodically send an AgentToServer
+message. At a minimum the instance_uid field MUST be set. It is recommended that the Agent
+also set ComponentHealth as well. An HTTP based-client MUST use the heartbeat interval as
+its polling interval.
+
+A heartbeat is used to keep a load balancer connection active AND inform the server that
```

Comment:
Same here, does "AND" need to be in caps?
```suggestion
A heartbeat is used to keep a load balancer connection active and inform the server that
```

Reply:
copied 😅